### PR TITLE
Emit necessary events

### PR DESCRIPTION
**Pull request by DemiMarie**
These had been TODOs in the code.

### CODE REVIEW

File: src/bridge.rs [hunk 1]
```diff
@@ -201,14 +201,10 @@ decl_module! {
...
                     if VoteThreshold::SuperMajorityApprove.approved(stake_sum, total_issuance - stake_sum, total_issuance, total_issuance) {
-                        match <balances::Module<T>>::decrease_free_balance(&tgt, qty) {
-                            Ok(_) => {
-                                // TODO: do we still mark completed on error? or store a "failed" tx?
-                                <WithdrawOf<T>>::insert(record_hash, (inx, tgt.clone(), qty, new_signers.clone(), true));
-                                // TODO: fire event
-                            },
-                            Err(err) => { return Err(err); } // TODO test this?
-                        };
+                         <balances::Module<T>>::decrease_free_balance(&tgt, qty)?;
```
Comment (drewstone) [inline src/bridge.rs]: Looks like the spacing is off by 1.

**Review by drewstone — COMMENTED** [review]
Looks good! Thanks for the PR!

**Review by drewstone — CHANGES_REQUESTED** [review]
Looks good! Thanks for the PR!

File: src/bridge.rs (hunk 1 above)
Comment (DemiMarie) [reply src/bridge.rs]: Thank you, fixed.

**Review by drewstone — APPROVED** [review]

**Pull request merged**